onlinejudge/_implementation/utils.py

```diff
@@ -50,6 +50,26 @@ def next_sibling_tag(tag: bs4.Tag) -> bs4.Tag:
     return tag
 
 
+# remove all HTML tag without interpretation (except <br>)
+# remove all comment
+# using DFS(Depth First Search)
+def parse_content(parent: Union[bs4.NavigableString, bs4.Tag, bs4.Comment]) -> bs4.NavigableString:
```

Comment:
Do you think this function can be used by any other contests in the near future?
If not, could you move it to yukicoder and make it class method?